pkg/urbit/vere/io/ames.c

```diff
@@ -1318,10 +1319,14 @@ _ames_io_talk(u3_auto* car_u)
   u3_ames* sam_u = (u3_ames*)car_u;
   _ames_io_start(sam_u);
 
-  // send born event
+  //  send born event
   //
   {
-    u3_noun wir = u3nt(c3__newt, u3k(u3A->sen), u3_nul);
+    //  XX remove [sev_l]
```

Comment:
Is this XX for now or later?

Reply:
It's for later; the question is just: is it desirable for these wires/ducts to change with each process lifetime? I tend to think not, but I'm not sure. (While I'm thinking of it, I should really get rid of `%newt` ...)